msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -980,17 +980,56 @@ The following table includes the query parameter:
 
 |Value|Type|Required|Description|
 |---|---|----|---|
-|**callback**| String | Yes | Callback contains two parameters `error` and `result`. The *error* can either contain an error type `SdkError` or `null` when the toggle is successful. The *result* can either contain true or false value, when the toggle is successful or null when the toggle fails. The incoming audio is muted if the result is true and unmuted if the result is false. |
+|**callback**| String | Yes | Callback contains two parameters `error` and `result`. The *error* can either contain an error type `SdkError` or `null` when the toggle is successful. The *result* can either contain true or false value, when the toggle is successful or null when the toggle fails. The incoming audio is muted if the result is true and unmuted if the result is false.
 
 ### Example
 
-```typescript
-function toggleIncomingClientAudio(callback: (error: SdkError | null, result: boolean | null) => void): void {
-    if (!callback) {
-      throw new Error('[toggle incoming client audio] Callback cannot be null');
+```Typescript
+callback = (error, result) => {
+        if (error) {
+            // Handle error code
+        }
+        else {
+            // Handle success code
+        }
+    }
+
+microsoftTeams.meeting.toggleIncomingClientAudio(this.callback)
+
+```
+
+> [!NOTE]
+> RSC has been enabled after 1.11, this feature won't work before 1.12.
+
+Use the following example to configure your app manifest's `webapplicationinfo` property for any private meeting:
+
+### For App manifest post version 1.12
+
+```JSON
+
+"webApplicationInfo": {
+    "id": "<bot id>",
+    "resource": "https://RSCPermission",
+    "applicationPermissions": [
+      "OnlineMeeting.ReadBasic.Chat"
+    ]
+}
+
+```
+
+### Sample Manifest
+
+```JSON
+
+"authorization": {
+    "permissions": {
+      "resourceSpecific": [
+        {
+          "name": "OnlineMeetingParticipant.ToggleIncomingAudio.Chat",
+          "type": "Delegated"
+        }
+      ]
```

Comment:
Added few more sections to update, so please check those as well and update in this PR.

Reply:
I have incorporated the suggested comments.